docs/Introduction/getting-started/fund-your-contract.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: nodes.liquid
+section: gettingStarted
+date: Last Modified
+title: "Fund Your Contracts"
+permalink: "docs/fund-your-contract/"
+---
+
+Some smart contracts require funding at their addresses so they can operate without you having to call functions manually and pay for the transactions through MetaMask. This guide explains how to fund Solidity contracts with LINK or ETH using the Remix IDE.
```

Comment:
You only ever fund like this if you use VRF, and that's going away. Should we just leave this page off of the getting started?

Reply:
Moved to the Ethereum Chains docs in the Developer Resources nav section.